Formula/dxpy.rb

```diff
@@ -50,6 +50,11 @@ class Dxpy < Formula
     sha256 "a0c30272fb4ddda5f5ffc1089d7405b7a71b0b0f51993cb4e5dbb4590b2fc229"
   end
 
+  resource "gnureadline" do
+    url "https://files.pythonhosted.org/packages/50/64/86085c823cd78f9df9d8e33dce0baa71618016f8860460b82cf6610e1eb3/gnureadline-6.3.8.tar.gz"
+    sha256 "58e208ed34a53456ecf58bc11eacbe332c50d322b329457e2c645bd15192b035"
+  end
```

Comment:
this is not needed.

Reply:
Why not? It's included in [`setup.py`](https://github.com/dnanexus/dx-toolkit/blob/d22239c6dddad2004cdc346ef2ba99840a7fc73d/src/python/requirements.txt#L9)

Reply:
because it is caused build issue, and I did include `readline` as separate dependency.

Reply:
looks like we have build auto-cancelled now, too bad that I could show what I said. I reset the commit to prove the point.

Reply:
That's fine (no need to prove it I was just curious)

Reply:
Can we put a comment somewhere to indicate that `gnureadline` shouldn't be added? `brew bump-formula-pr` will probably add it each time.

Reply:
Sounds good, always to have this knowledge somewhere is good :)

Reply:
adding a comment now.